Misc/NEWS.d/next/Library/2025-03-23-18-39-07.gh-issue-60115.AWdcmq.rst

```diff
@@ -0,0 +1 @@
+Support frozen modules for :func:`linecache.getline`.
```

Comment:
Actually, I think we didn't document `getlines` so you can write 

```rst
:func:`!linecache.getlines`
```

instead.

Reply:
I realized that as well. We can talk about the public interface. `linecache.getline` also benefits from this change.